sph/test/hydro_ve/av_switches_kern.cpp

```diff
@@ -84,7 +84,7 @@ TEST(AVswitches, JLoop)
 
     for (i = 0; i < neighborsCount + 1; i++)
     {
-        kx[i] = K * xm[i] / ::sphexa::math::pow(h[i], 3);
+        kx[i] = K * xm[i] / math::pow(h[i], 3);
```

Comment:
Why is not this pow "sph::math::pow" ?

Reply:
that's what it really is. we're already inside namespace `sph`.